### ISSUE TITLE
[FEATURE] Smooth progress bar update

### ISSUE THREAD

**Issue by vanfresh** [jhomlala/betterplayer#273]
**Is your feature request related to a problem? Please describe.**
The progress bar update isn't smooth.  It jumps positions about every second.  This is fine for long videos, but looks glitchy on shorter videos.

**Describe the solution you'd like**
I would like the progress bar to update smoothly. 

**Describe alternatives you've considered**
I tried tweening the progress bar but comes with an obvious lag even if I account for timing between each update.

**Comment by jhomlala** [comment]
Won't do that. To update progress bar, BetterPlayer need to access native player code to access position and duration. Right now delay between accessing values is 500 ms and it's enough. Accessing it more frequently imo may lead to lag and janks. Anyway you can do fork and change value. Check this line: https://github.com/jhomlala/betterplayer/blob/master/lib/src/video_player/video_player.dart#L448
Simply put smaller value here. 

Also I've checked youtube app and progress bar also has jumps.
